src/components/Theme/style.css

```diff
@@ -32,6 +29,11 @@
 .content {
   flex: 1 0 auto;
   position: relative;
+  display: flex;
+
+  > * {
+    flex-grow: 1;
```

Comment:
is it safe to apply this to all of the children? Was it the goal of the instruction?

Reply:
I don't think so, but would be happy use an object mapper and add it to utils if you think it's useful.

Reply:
The reason for this change is:
> All actions should sit at the bottom of the box on mobile

We need the content to be of `display: flex` in order to take the full height of its parent. Previously it wasn't `flex`, and in order to take the full height the approach was to absolutely position the child holding the copy. This approach will work, but it won't push siblings to the bottom, causing some overlapping issues like the one described [https://github.com/onfido/onfido-sdk-ui/pull/349#issuecomment-401108985](here).

Also described in #356 and [https://github.com/onfido/onfido-sdk-ui/pull/349#issuecomment-399902699](here).

It should be safe to apply to all first immediate children of `.content` (it's always one):
-by default it will be `flex-direction: row`, and only